sonic-ledd/scripts/ledd

```diff
@@ -76,7 +76,7 @@ class DaemonLedd(DaemonBase):
             sys.exit(LEDUTIL_LOAD_ERROR)
 
         # Open a handle to the Application database
-        appl_db = daemon_base.db_connect(swsscommon.APPL_DB)
```

Comment:
why not use swsscommon.APPL_DB?

Reply:
swsscommon.APPL_DB is db id from swsscommon c++ library. Here we need a string "APPL_DB", For backward compatibility, we leave db id still there and not changed APPL_DB = "APPL_DB"